riptide-spring-boot-autoconfigure/README.md

```diff
@@ -129,6 +130,18 @@ Required when `transient-fault-detection` is enabled.
 </dependency>
 ```
 
+#### [Compression](../riptide-compression) support
+
+Required when `request-compression` is enabled.
```

Comment:
As it's a dependency-free module, we may want to mark it as non-optional...

Reply:
Sounds good

Reply:
Adapted.